tests/suites/helpers.function

```diff
@@ -300,6 +302,10 @@ jmp_buf jmp_tmp;
 #define ENTROPY_HAVE_STRONG
 #endif
 
+/* Set flag if number of entropy sources is more than 2 */
+#if ( MBEDTLS_ENTROPY_MAX_SOURCES > 2 )
+#define ENTROPY_SOURCE_COUNT_MORE_THAN_TWO
```

Comment:
Minor: fine for `baremetal` but for upstreaming, I'd like a better name than `ENTROPY_SOURCE_COUNT_MORE_THAN_TWO`.  Maybe `MBEDTLS_SINGLE_ENTROPY_SOURCE`, and then check for that to *not* be set.